showcase/src/app/welcome/pages/reactive-form-errors/demo-reactive-form-errors-page.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { AbstractControl, FormBuilder, FormControl, FormGroup, Validators } from "@angular/forms";
+import { ErrorStateMatcher } from "@angular/material/core";
+import { ParentErrorStateMatcher } from "./parent-error-state-matcher";
+import { PasswordValidator } from "./password-validator";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {ReferenceLink} from '../../../shared/components/reference-block';
+@Component({
+	selector: "showcase-reactive-forms-with-ngx-form-errors",
+	templateUrl: "./demo-reactive-form-errors-page.component.html",
+	styleUrls: ["./demo-reactive-form-errors-page.component.scss"]
+})
+export class DemoReactiveFormErrorsPageComponent implements OnInit {
+	public collapsed: boolean[] = [false, false, true];
+
+	public referenceList: ReferenceLink[] = [
+		{
+			label: "NGX Form errors repository",
```

Comment:
I think it is better to use the word `library` instead of `repository` so it is generic enough cause maybe we will point to a different page in the future (docs or demo page, who knows) ;)

Reply:
Ok, so I can phrase it this way : Reactive form errors library ?

Reply:
I would say: `NGX Form erros library` 😉

Reply:
okok